crates/kitsune_p2p/kitsune_p2p/src/types.rs

```diff
@@ -31,6 +31,10 @@ pub enum KitsuneP2pError {
     /// Other
     #[error("Other: {0}")]
     Other(Box<dyn std::error::Error + Send + Sync>),
+
+    /// Reqwest crate.
+    #[error(transparent)]
+    Reqwest(#[from] reqwest::Error),
```

Comment:
```suggestion
    /// Reqwest crate.
    #[error(transparent)]
    Reqwest(#[from] reqwest::Error),
    
    /// Other
    #[error("Other: {0}")]
    Other(Box<dyn std::error::Error + Send + Sync>),
```

just... personal preference to keep the catch-all generic error type at the bottom : )